addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -112,25 +112,19 @@ options.registry.gallery = options.Class.extend({
             save: images => {
                 // TODO In master: restore addImages Promise result.
                 this.trigger_up('snippet_edition_request', {exec: () => {
-                    let $newImageToSelect;
```

Comment:
Do not hesitate to tell me if you prefer me to do this remove of the unused variables in the master version.

Reply:
It's a local variable, nobody can use it except this very codebase here... there is no reason to hesitate to keep it in stable :grimacing: